lib/xdr/contract/spec/sc_spec_entry.ex

```diff
@@ -0,0 +1,79 @@
+defmodule StellarBase.XDR.SCSpecEntry do
+  @moduledoc """
+  Representation of Stellar `SCSpecEntry` kind.
+  """
+
+  alias StellarBase.XDR.{
+    SCSpecEntryKind,
+    SCSpecFunctionV0,
+    SCSpecUDTStructV0,
+    SCSpecUDTUnionV0,
+    SCSpecUDTEnumV0,
+    SCSpecUDTErrorEnumV0
+  }
+
+  @behaviour XDR.Declaration
+
+  @arms [
+    SC_SPEC_ENTRY_FUNCTION_V0: SCSpecFunctionV0,
+    SC_SPEC_ENTRY_UDT_STRUCT_V0: SCSpecUDTStructV0,
+    SC_SPEC_ENTRY_UDT_UNION_V0: SCSpecUDTUnionV0,
+    SC_SPEC_ENTRY_UDT_ENUM_V0: SCSpecUDTEnumV0,
+    SC_SPEC_ENTRY_UDT_ERROR_ENUM_V0: SCSpecUDTErrorEnumV0
+  ]
+
+  @type code ::
+          SCSpecFunctionV0.t()
+          | SCSpecUDTStructV0.t()
+          | SCSpecUDTUnionV0.t()
+          | SCSpecUDTEnumV0.t()
+          | SCSpecUDTErrorEnumV0.t()
+
+  @type kind :: SCSpecEntryKind.t()
+
+  @type t :: %__MODULE__{code: code(), kind: kind()}
+
+  defstruct [:code, :kind]
```

Comment:
Nice catch! The `kind` field is OK👌🏻 

Please, update the SCEnvMetaEntry module in this PR to rename the struct field from `type` to `kind`